.jenkins/Jenkinsfile

```diff
@@ -271,7 +295,25 @@ def ACCHostVerificationTest(String version, String build_type) {
             }
         }
 
-        /* TODO: Windows nonSGX stage.  Waiting for windows non-SGX VM */
+        /* Windows nonSGX stage. */
+        stage("Windows nonSGX Verify Quote") {
+            node("nonSGX-Windows") {
+                timeout(GLOBAL_TIMEOUT_MINUTES) {
+                    cleanWs()
+                    checkout scm
+                    unstash "linux_host_verify-${build_type}-${BUILD_NUMBER}"
+                    powershell 'Copy-Item -Recurse C:\\openenclave\\prereqs\\nuget ${env:WORKSPACE}\\prereqs'
+                    dir('build') {
+                        bat """
+                            vcvars64.bat x64 && \
+                            cmake.exe ${WORKSPACE} -G Ninja -DBUILD_ENCLAVES=OFF -DUSE_LIBSGX=OFF -DCMAKE_BUILD_TYPE=${build_type} -Wdev && \
+                            ninja -v && \
+                            ctest.exe -V -C ${build_type} -R host_verify --output-on-failure --timeout ${CTEST_TIMEOUT_SECONDS}
```

Comment:
This is really quite nice work figuring this out, Sergio. Nice job!

Reply:
Thanks!